.github/workflows/scripts/helpers/publish-packages-helper.mjs

```diff
@@ -29,10 +29,8 @@ export const getPackagesToPublish = async () => {
 
 /**
  * Releases to npm using changeset publish.
- * @param {string} pkg
- * @param {string} version (E.g. "3.6.0")
  */
-export const createNpmRelease = async (pkg, version) => {
+export const createNpmRelease = async () => {
```

Comment:
This change made me realize we don't need to call `createNpmRelease` multiple times (in a loop).

Reply:
Ohh, right, moved out of the loop (https://github.com/iTwin/iTwinUI/pull/1920/commits/5905b5833b1b408356d48c1f9d07c819aaee79a2).